### PR TITLE
feat: update homepage description

### PR DESCRIPTION
**Pull request by BastiDood**
This PR updates the homepage description for the website as suggested by President @calebbun.

Cc: @Anjellyrika (Director for Branding and Creatives)
Cc: @beyatris (Director for External Relations)
Cc: @allainerain (Head of Content for the Engineering Committee)

I would like to know your thoughts/objections first before I merge this in.

### CODE REVIEW

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -7,22 +7,41 @@
...
+        </li>
+    </ul>
+</section>
+<section class="prose max-w-2xl max-w-full p-8 dark:prose-invert sm:p-12 md:p-16">
+    <h1 class="font-heading">Our Vision</h1>
     <ul>
-        <li>Enhancing and upskilling student abilities through our developer training program.</li>
-        <li>Exploring avenues for industry partnerships.</li>
-        <li>Fulfilling the needs of the community through technological innovation.</li>
-        <li>Providing solutions to <em>your</em> problems.</li>
```
Comment (Anjellyrika) [inline src/routes/+page.svelte:25]: I would like to keep these lines in some way, although not necessarily under the "Our Vision" section. These lines with the keywords "partnerships", "fulfilling needs", and usage of the second-person "you" , then directly followed by the contact form when scrolling downwards presents more appeal to prospective clients, in my opinion. :D

Comment (BastiDood) [reply src/routes/+page.svelte:25]: Cc: @calebbun

What do you think?

File: src/routes/+page.svelte [hunk 2]
```diff
@@ -7,22 +7,41 @@
 
 <section class="prose max-w-2xl p-8 dark:prose-invert sm:p-12 md:p-16">
     <h1 class="font-heading">Learn. Create. <em>Innovate.</em></h1>
+    <h2 class="font-heading">University of the Philippines Center for Student Innovations</h2>
     <p>
-        UP CSI is a service-oriented student organization aimed towards the enhancement of student
-        learning by working with real-world software projects.
+        A service-oriented organization aimed towards the enhancement of student learning from the
```
Comment (calebbun) [inline src/routes/+page.svelte:12]: Can we change this into "We are a service-oriented ..."

File: src/routes/+page.svelte (hunk 1 above)
Comment (BastiDood) [reply src/routes/+page.svelte]: As a resolution, I have reinstated the **What We Do** section below **Our Vision**.

**Review by calebbun — APPROVED** [review]
Looks good!

**Review by Anjellyrika — APPROVED** [review]
LGTM!

**Pull request merged**